TypeCobol/Compiler/Directives/TypeCobolOptions.cs

```diff
@@ -55,6 +60,7 @@ public TypeCobolOptions(TypeCobolConfiguration config)
             AutoRemarksEnable = config.AutoRemarks;
 #endif
             CheckEndAlignment = config.CheckEndAlignment;
+            this.IsCobolLanguage = config.IsCobolFile;
```

Comment:
I'd rather use the same name in TypeCobolOptions and TypeCobolConfiguration
```suggestion
            this.IsCobolLanguage = config.IsCobolLanguage ;
```

Reply:
done in cf3b16c